release-notes.html.md.erb

```diff
@@ -5,6 +5,23 @@ owner: PCF Healthwatch
 
 ##<a id='releases'></a>Releases
 
+###<a id='v1.4.6'></a>v1.4.6
+
+**Release Date: May CHANGEME, 2019**
+
+* **[Feature]** Improve troubleshooting of [BOSH Director Health SLI](./metrics.html#bosh-director) failures via logging in `bosh-health-check` app
+* **[Bug Fix]** Fixed alerting on Isolation Segments where the alert was incorrectly associated to the `cf` deployment
```

Comment:
TODO for docs review: this bug actually affects any SVM that associated with the isolation segment. It will display the red stoplight on the CF stoplight and green stoplight on the Isolated stoplight where it should be the opposite way. Is the current description good enough?

Reply:
Resolved with @paigecalvert